pkg/ccl/backupccl/full_cluster_backup_restore_test.go

```diff
@@ -777,10 +776,13 @@ func TestClusterRestoreFailCleanup(t *testing.T) {
 				{"zones"},
 			},
 		)
+
+		close(blockCh)
```

Comment:
could you still use defer here, since you're moving this channel declaration into a function definition?